plugins/package-managers/node/src/funTest/assets/projects/synthetic/npm/README.md

```diff
@@ -1,26 +0,0 @@
-# npm
```

Comment:
Please still provide a rational in the commit message why it's ok to drop the contents of the file and why no useful information is lost.

Reply:
Hint: Maybe it's easier to find an argument if you move the last commit to before this commit, and first add *all* links in this file to the respective places in the `NodePackageManager` enum. Then you could say that this file just describes the standard layout we have, which is somewhat self-explaining from the directory names, and that the links are by now maintained as part of class docs.

Reply:
I tried to find a rationale for only keeping the shrinkwrap link

Reply:
> The other links are only for lockfile documentation.
As lockfile is common terminology and the links can be found easily,
it's not necessary to keep them around.

But the shrinkwrap file also is just a lockfile format... the [linked docs](https://docs.npmjs.com/cli/v6/configuring-npm/shrinkwrap-json) explicitly state:

> It is identical to `package-lock.json`, with one major caveat

So the current argumentation does not really hold: You should either keep all links to lockfile documentation, or none. I'd prefer to keep them all for easy lookup, and to avoid finding an argument to remove them.

Reply:
> But the shrinkwrap file also is just a lockfile format... the [linked docs](https://docs.npmjs.com/cli/v6/configuring-npm/shrinkwrap-json) explicitly state:

I didn't know this was a lockfile and it wasn't obvious at all from the name to me. 

> So the current argumentation does not really hold:

I believe it does for above reason.

> I'd prefer to keep them all for easy lookup, and to avoid finding an argument to remove them.

If you still want these, what do you think would be a good place to put them ?

Reply:
Well, as [mentioned](https://github.com/oss-review-toolkit/ort/pull/9158#discussion_r1763176079) I would simply put the links next to corresponding `NodePackageManager` entries in the same way you did for shrinkwrap. So like:

    NPM(
        lockfileName = "package-lock.json", // See https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json.
        markerFileName = "npm-shrinkwrap.json" // See https://docs.npmjs.com/cli/v6/configuring-npm/shrinkwrap-json.
    ) {

    // ...

    PNPM(
        lockfileName = "pnpm-lock.yaml", // See https://pnpm.io/git#lockfiles.
        workspaceFileName = "pnpm-workspace.yaml"
    ) {

    // ...

    YARN(
        lockfileName = "yarn.lock" // See https://classic.yarnpkg.com/en/docs/yarn-lock.
    ) {

    // ..

    YARN2(
        lockfileName = "yarn.lock", // See  // See https://classic.yarnpkg.com/en/docs/yarn-lock..
        markerFileName = ".yarnrc.yml"
    ) {

Note that I've also added a link for PNPM's lockfile although it was not present in the `README.md` before for consistency.